core-bundle/src/Security/Voter/MemberGroupVoter.php

```diff
@@ -25,19 +26,37 @@ protected function supports($attribute, $subject): bool
             return false;
         }
 
-        return is_numeric($subject)
-            || (\is_array($subject) && \count($subject) === \count(array_filter($subject, 'is_numeric')));
+        if (!\is_array($subject)) {
+            return (string) (int) $subject === (string) $subject;
+        }
+
+        $filtered = array_filter($subject, static function ($val) { return (string) (int) $val === (string) $val; });
+
+        return \count($subject) === \count($filtered);
```

Comment:
I think we should remove all of this logic, because if `ContaoCorePermissions::MEMBER_IN_GROUPS === $attribute` is true, the voter supports voting on the subject, no matter what the subject is. We will filter non-numeric values later in the `voteOnAttribute()` method.

Reply:
fine with me 😊